docs/en_us/platform_api/source/mobile/users.rst

```diff
@@ -1,10 +1,8 @@
-#########################
-User API Module
-#########################
+################################
+Platform Mobile User API Module
```

Comment:
Same comment re headings as above. Should this be:
"EdX Platform Mobile API User Module"?